manifests/poc-data-cluster/prod/spark/iot-packets-silver.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: iot-packets-silver-query
+  namespace: spark
+data:
+  query.sql: |
+    SELECT 
+      gateway_tmst,
+      oui,
+      net_id,
+      rssi,
+      frequency,
+      snr,
+      datarate,
+      region,
+      b58encodeChecked(gateway) as gateway_key,
+      payload_hash,
+      payload_size
+      free,
+      type,
+      received_timestamp,
+      date,
+      file
+    FROM packet_router_packet_report_v1
+
+---
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: iot-poc-silver
+  namespace: spark
+spec:
+  type: Scala
+  mode: cluster
+  image: "public.ecr.aws/k0m1p4t7/spark:v3.4.0-aws"
+  imagePullPolicy: Always
+  mainClass: Main
+  mainApplicationFile: "s3a://foundation-data-lake-requester-pays/jars/spark-streaming-sql-assembly-1.0.jar"
+  sparkVersion: "3.4.0"
+  restartPolicy:
+    type: Never
+  sparkConf:
+    spark.databricks.delta.autoCompact.enabled: "true"
+  hadoopConf:
+    fs.s3a.aws.credentials.provider: com.amazonaws.auth.WebIdentityTokenCredentialsProvider
+  volumes:
+    - name: "tmp"
+      hostPath:
+        path: "/tmp"
+        type: Directory
+    - name: config-vol
+      configMap:
+        name: iot-poc-silver-query
+        items:
+          - key: query.sql
+            path: query.sql
+  driver:
+    serviceAccount: spark-data-lake-access
+    cores: 1
+    coreLimit: "1200m"
+    memory: "512m"
+    nodeSelector:
+      node.kubernetes.io/instance-type: m5.large
+    envVars:
+      TABLE_LORA_POC_V1: s3a://foundation-data-lake-requester-pays/bronze/packet_router_packet_report_v1
```

Comment:
Env var should be `TABLE_PACKET_ROUTER_PACKET_REPORT_V1` if that's how you want it to be named in your select statement